migrations/00000-createTableGuests.ts

```diff
@@ -0,0 +1,23 @@
+import { Sql } from 'postgres';
+
+export type Guest = {
+  id: number;
+  firstName: string;
+  lastName: string;
+  attending: boolean;
+};
+
+export async function up(sql: Sql) {
+  await sql`
+    CREATE TABLE guests (
+      id integer PRIMARY key generated always AS identity,
```

Comment:
is `key`, `generated`, etc supposed to be lowercase here? 

it seems like it was formatted as uppercase in the Next.js example:

- https://github.com/upleveled/next-js-example-spring-2024-atvie/blob/main/migrations/00000-createTableAnimals.ts#L22